data/avgi/avgi 1 first contact.txt

```diff
@@ -0,0 +1,834 @@
+# Copyright (c) 2023 by Azure
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+
+mission "Avgi: Trapped in Twilight"
+	invisible
+	landing
+	to complete
+		never
+	on enter
+		system
+			attributes "twilight" "tangled shroud"
+		fail
+		set "avgi: twilight trapped"
+		dialog phrase "twilight trapped"
+
+phrase "twilight trapped"
+	word
+		`As soon as you jump into the system, you notice something is wrong. The characteristic plasma of your jump drive fades unnaturally quickly, and as you check your ship's systems, you notice its temperature is somehow much lower than normal. You attempt to plot a jump back to human space, but your computer tells you the system you just came from is out of range. It appears you are stuck in this region for now.`
+
+mission "Avgi: First Contact"
```

Comment:
I assume this is intentional- Distant Shores vouching for you, perhaps- but you can totally land on Aviskir and get refueled/repaired. It's a little funny after having read the mission here to not have to worry about battery fire....